### DISCUSSION TITLE
Would microsoft openjdk support macOS ARM?

### DISCUSSION THREAD

**Discussion by gtn1024** [microsoft/openjdk#38] [Q&A] [answered]
Given there is only [Azul Zulu JDK](https://www.azul.com/downloads/zulu-community) offer the version of macOS ARM, would microsoft openjdk support the new Mac?

![image](https://user-images.githubusercontent.com/69113276/116103861-8dfe3900-a6e2-11eb-81b4-491cafed8b2a.png)

**Comment by gtn1024** [comment, marked as answer]
I've found the post before, sorry...
[AArch64 binaries for Apple Silicon?](https://github.com/microsoft/openjdk/discussions/7)

**Comment by aluneed** [comment]
[Liberica](https://bell-sw.com/pages/downloads/) is an another choice at present.
